dwave/system/testing.py

```diff
@@ -20,23 +20,23 @@
 import dwave_networkx as dnx
 from tabu import TabuSampler
 import dwave.cloud.computation
+import warnings
 
 try:
-    from neal import SimulatedAnnealingSampler
+    #Simplest and efficient choice returning low energy states:
+    from greedy import SteepestDescentSampler as SubstituteSampler
+    mock_fallback_substitute = False
 except ImportError:
-    from dimod import SimulatedAnnealingSampler
-
+    from dimod import SimulatedAnnealingSampler as SubstituteSampler
+    mock_fallback_substitute = True
```

Comment:
I don't think this is necessary, you can just do `if SubstituteSampler is not dimod.SimulatedAnnealingSampler` below.

Reply:
❤️